flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonRowDataDeserializationSchema.java

```diff
@@ -101,6 +102,9 @@ public JsonRowDataDeserializationSchema(
 	public RowData deserialize(byte[] message) throws IOException {
 		try {
 			final JsonNode root = objectMapper.readTree(message);
+			if (root instanceof MissingNode) {
```

Comment:
I think this is a little hack. This can't handle nested types. Would be better fix this in `org.apache.flink.formats.json.JsonToRowDataConverters#wrapIntoNullableConverter`. And call `JsonToRowDataConverters#createConverter` in the constructor.

Reply:
Thanks for your suggestion,i'll update it